.github/ISSUE_TEMPLATE/convergence_epic.md

```diff
@@ -1,48 +1,55 @@
 ---
-name: (internal) Component convergence epic
-about: (team member use only) Epic issue tracking convergence of a particular component
+name: (internal) Component implementation epic
+about: (team member use only) Epic issue tracking implementation of a particular component
 ---
 
 <!--
 These issues are used by core contributors to track the list of items that should be
-completed as part of converging a component. More info can be found here: https://github.com/microsoft/fluentui/wiki/Component-Convergence-Guide
+completed as part of creating a v9 component. More info can be found here: https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide
 -->
 
-## Preparation:
+## Preparation
 
-- [ ] Open UI Research
+- [ ] [Open UI Research](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#open-ui-research)
   - [link to https://open-ui.org/]
-- [ ] Open GitHub issues related to component
+- [ ] [Open GitHub issues related to component](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#find-open-issues-on-github)
   - [link to each issue]
-- [ ] react-\* package scaffolded with the right ownership in CODEOWNERS
-  - [link to package / PR]
-- [ ] Component Spec authored and reviewed
-  - [link to spec in component package / PR]
+- [ ] [Create react-\* package and component from template](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#component-package)
+  - [link to PR]
+  - [link to package: `https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-<your-component>`]
+- [ ] (Optional) [Draft implementation](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#draft-implementation)
+  - [link to PR, if applicable]
```

Comment:
@GeoffCox had a good idea about a codesandbox template for the Draft Implementation. Don't have to block on this, but I wanted to mention him so he saw this as a good place to put the link/idea bout the template

Reply:
That's a good idea! I don't think this issue tempalte needs to mention the details specifically, but @GeoffCox it would be great to add details about that to the wiki article linked from here: https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#draft-implementation

Reply:
I guess my main feedback here would be to actually encourage folks to use a CodeSandbox instead of a PR. So this:

```suggestion
- [ ] (Optional) [Draft implementation](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#draft-implementation)
  - [link to CodeSandbox, if applicable]
```

Reply:
I think I'll make it more generic of "link to draft implementation, if applicable". That way, people can implement the draft in whatever way makes the most sense to them. I think it's worthwhile updating the wiki article to suggest using CodeSandbox, but it's better to leave it unspecified in the epic issue. Does that sound good?